less/mixins.less

```diff
@@ -1 +1,37 @@
-// Mixins Here
\ No newline at end of file
+// Mixins Here
+.buttons(@width; @bgc; @tc; @fontsize) {
+	background-color: @bgc;
+	color: @tc;
+	border-radius: 12px;
+	font-size: @fontsize;
+	text-align: center;
+    vertical-align: middle;
+    opacity: 0.6;
+    transition: 0.6s;
```

Comment:
You can also add your `hover` psuedo selector, and even mobile breakpoint here so that you only need to call this mixin on any button you add, without having to worry about adding any other styles.